static_src/util/cf_api.js

```diff
@@ -28,6 +51,36 @@ function handleError(err, errHandler = errorActions.errorFetch) {
   }
 }
 
+function parseError(resultOrError) {
```

Comment:
@msecret I'm still not quite sold on how we want to represent errors in the app. Because of the CF API, we expect most errors to have a `description` property, but most JS errors will only have `message` and we don't have consistent handling to move `description` to `message` or vice versa.

I'm starting to think we should just expect the standard `message`, and make sure to fill in something descriptive based on the specific error type we have. Kind of like how [this](https://github.com/18F/cg-dashboard/pull/1054/commits/6d6cc3b74b92a2dbdf0fcfb791b39ecb2f7e387e#diff-3fd0a313bfc0cb4679cc0af796e67a94R15) works, but to populate `message` instead.

I'm a fan of using custom error types (extending `Error`) so it's easier to change the error handling based on the type of error (using `instanceof Type` checks) but babel makes it a little awkward--you can't use `class extends` with native types apparently.

Any other thoughts?